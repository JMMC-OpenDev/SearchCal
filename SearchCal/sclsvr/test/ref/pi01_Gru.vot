version https://git-lfs.github.com/spec/v1
oid sha256:a5da6e2761234f90678bd4fbeae275edc9f73483e4b06231f4dc4e907160b7cc
size 157652
