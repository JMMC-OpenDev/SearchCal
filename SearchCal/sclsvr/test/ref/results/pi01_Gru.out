version https://git-lfs.github.com/spec/v1
oid sha256:f5f31718b23b98f00851651da386ba2937f93837078abababf5afc1209f36549
size 230227
